test-suite/src/data_type/int128.rs

```diff
@@ -0,0 +1,106 @@
+use {
+    crate::*,
+    gluesql_core::{data::ValueError, prelude::DataType, prelude::Value::*},
+};
+
+test_case!(int128, async move {
+    run!(
+        "CREATE TABLE Item (
+        field_one INT(128),
+        field_two INT(128),
+    );"
+    );
+    run!("INSERT INTO Item VALUES (1, -1), (-2, 2), (3, 3), (-4, -4);");
+
+    let parse_i128 = |text: &str| -> i128 { text.parse().unwrap() };
+
+    let max_str = "170141183460469231731687303715884105728";
+    let min_str = "-170141183460469231731687303715884105729";
```

Comment:
I saw that the comment which describes these values are located in the below.
It would be better for attaching comments to here.
It would be also good to use `const`
```rust
// i128::MAX + 1
const MAX_STR: &str = "170141183460469231731687303715884105728";

// i128::MIN - 1
const MIN_STR: &str = "-170141183460469231731687303715884105729";
```